roles/var_add_files/tasks/main.yml

```diff
@@ -1,6 +1,17 @@
 ---
 # vim: set ft=ansible:
 #
+- name: Set fedora25 to be False by default
+  set_fact:
+    fedora25: False
+
+- name: Set fedora25 to be True for F25
+  set_fact:
+    fedora25: True
+  when:
+    - ansible_distribution == "Fedora"
+    - ansible_distribution_version == "25"
```

Comment:
Given it's highly likely at some point this gets downstream, I'd try to detect it - something like run `chcon -t container_file_t foo` and see whether or not that works.